README.md

```diff
@@ -10,14 +11,55 @@ If a member has made a payment in the past 30 days, they will be considered an a
 - the server will maintain access lists and periodically push those access lists to the microcontrollers on the network
 - The microcontroller (aka a resource) stores its access list locally so it's not dependant on the network when someone wants to access the space
 
-## Install prereqs
+# Dev Setup
+
+## The Easy Way (using Remote-Containers)
+Install the following
+- [Docker](https://www.docker.com/products/docker-desktop)
+- [VS Code](https://code.visualstudio.com/download)
+- VS Code [Remote-Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension 
+
+That's it! Nothing else needs to be installed.  
+
+If you are on Windows, then it is recommended to clone to the repo to a folder on the WSL for best performance.  
+
+Either open the folder in VS Code and choose reopen in container or run the Remote-Containers: Open Folder in Container... command and select the local folder.
+
+![Open from container](openFromContainer.gif "Open from container")
+
+
+The backend server can be started by debugging in VS Code. Otherwise, you can start the server without debugging using `sh buildandrun.sh`.
+
+Start the react app as described in the [UI README](/ui/README.md).
```

Comment:
Would it be alright if you could change this to `Start the web app`? This isn't a React app

Reply:
LOL!  I hadn't looked that closely at the web app yet.  I thought I was just out of date with my react knowledge!  This is actually using Material web components, right?

Reply:
Yup